chaoswm/utils.py

```diff
@@ -34,9 +38,10 @@ def get_wm_params(c: Dict[str, Any]) -> Optional[Dict[str, Any]]:
     return {"url": url, "timeout": timeout}
 
 
-def check_configuration(c: Dict[str, Any] = None) -> bool:
-    c = c or {}
-    if "wiremock" not in c:
+def check_configuration(configuration: Dict[str, Any] = None) -> bool:
+    """Check configuration contains valid wiremock settings"""
+    configuration = configuration or {}
+    if "wiremock" not in configuration:
         logger.error("Error: wiremock key not found in configuration section")
         return False
     return True
```

Comment:
Interesting. Why would we not fail the action with `ActivityFailed` in this case?

Reply:
In the original module this just delegates the error handling to the caller which is currently just nicely return empty responses. We avoided modifying anything that wasn't necessary to keep the PR as small as possible but failing seems nicer to me too 👍 

I will change this and re-run all our experiments to see if this change could break existing templates

Reply:
That's fine, you can leave it as-is. We can always make a more specific PR in the future for this.